### ISSUE TITLE
Error when upgrading/updating from commerce 1.

### ISSUE THREAD

**Issue by lukeholder** [craftcms/commerce#423]
### Description

Error: Database Exception: SQLSTATE[01000]: Warning: 1265 Data truncated for column 'orderLocale' at row 

```
Database Exception: SQLSTATE[01000]: Warning: 1265 Data truncated for column 'orderLocale' at row 1
The SQL being executed was: ALTER TABLE `craft_commerce_orders` CHANGE `orderLocale` `orderLocale` varchar(12) NOT NULL

Migration: craft\commerce\migrations\m180601_161904_fix_orderLanguage

Output:

    > alter column orderLocale in table {{%commerce_orders}} to string(12) NOT NULL ...Exception: SQLSTATE[01000]: Warning: 1265 Data truncated for column 'orderLocale' at row 1
The SQL being executed was: ALTER TABLE `craft_commerce_orders` CHANGE `orderLocale` `orderLocale` varchar(12) NOT NULL (/Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/db/Schema.php:664)
#0 /Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/db/Command.php(1263): yii\db\Schema->convertException(Object(PDOException), 'ALTER TABLE `cr...')
#1 /Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/db/Command.php(1075): yii\db\Command->internalExecute('ALTER TABLE `cr...')
#2 /Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/db/Migration.php(420): yii\db\Command->execute()
#3 /Users/luke/Documents/development/floom-3/vendor/craftcms/commerce/src/migrations/m180601_161904_fix_orderLanguage.php(27): yii\db\Migration->alterColumn('{{%commerce_ord...', 'orderLocale', Object(craft\db\mysql\ColumnSchemaBuilder))
#4 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/db/Migration.php(56): craft\commerce\migrations\m180601_161904_fix_orderLanguage->safeUp()
#5 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/db/MigrationManager.php(243): craft\db\Migration->up(true)
#6 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/db/MigrationManager.php(163): craft\db\MigrationManager->migrateUp(Object(craft\commerce\migrations\m180601_161904_fix_orderLanguage))
#7 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/services/Updates.php(216): craft\db\MigrationManager->up()
#8 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/controllers/BaseUpdaterController.php(435): craft\services\Updates->runMigrations(Array)
#9 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/controllers/UpdaterController.php(207): craft\controllers\BaseUpdaterController->runMigrations(Array, 'restore-db')
#10 [internal function]: craft\controllers\UpdaterController->actionMigrate()
#11 /Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#12 /Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#13 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/web/Controller.php(103): yii\base\Controller->runAction('migrate', Array)
#14 /Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('migrate', Array)
#15 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/web/Application.php(282): yii\base\Module->runAction('updater/migrate', Array)
#16 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/web/Application.php(652): craft\web\Application->runAction('updater/migrate')
#17 /Users/luke/Documents/development/floom-3/vendor/craftcms/cms/src/web/Application.php(236): craft\web\Application->_processUpdateLogic(Object(craft\web\Request))
#18 /Users/luke/Documents/development/floom-3/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#19 /Users/luke/Documents/development/floom-3/web/index.php(20): yii\base\Application->run()
#20 /Users/luke/.composer/vendor/laravel/valet/server.php(147): require('/Users/luke/Doc...')
#21 {main}

```
### Steps to reproduce

1. Update Craft and Commerce

**Comment by lukeholder** [comment]
This is due to missing `commerce_order` table data in the `orderLocale` column.

**Comment by lukeholder** [comment]
Fixed in 0fd8cc3be0bfbd16b486b94076b2693702285411
